docs/articles/testkube-dashboard-explore.md

```diff
@@ -1,44 +1,51 @@
-# The Testkube Dashboard
+# Testkube Dashboard
 
-The Testkube Dashboard provides a centralized user interface for managing your Testkube deployment. The Dashboard
-is included in the Testkube Control Plane and available after installation either via local port-forwarding using the 
-`testkube dashboard` command or via the NGINX Ingress Controller - [Read More](/articles/install/install-with-helm). 
+The Testkube dashboard offers a centralized user interface for managing Testkube deployments. The dashboard is part of the 
+Testkube Control Plane and is available after installation via local port-forwarding using the 
+`testkube dashboard` command or through use of the NGINX Ingress Controller. For more information, 
+read [Production Installations](/articles/install/install-with-helm). 
 
-When opening the Dashboard you will be presented with the following layout:
+**Dashboard**:
 
-![Testkube Dashboard](../img/dashboard-072024.png)
+![Testkube Dashboard](../articles/images/dashboard-overview.png)
 
-The toggles at the top of the screen (1) allow you to choose the Organization and Environment 
-currently shown in the Dashboard.
+**Testkube Sandbox** appears in the **Organization** and **Environment** drop-down list boxes near the dashboard (top left). 
```

Comment:
nitpick: suggest to merge these two sentences; 

**Testkube Sandbox** appears in the **Organization** and **Environment** drop-down list boxes near the dashboard (top left) when you are in the Testkube Sandbox environment.

Reply:
also - add a [Read More] link going to https://docs.testkube.io/articles/environment-management#testkube-sandbox-environment (which in itself needs to be updated a bit..)